passageidentity.gemspec

```diff
@@ -19,12 +19,8 @@ Gem::Specification.new do |s|
   s.metadata['source_code_uri'] = 'https://github.com/passageidentity/passage-ruby'
```

Comment:
The homepage link doesn't work on https://rubygems.org/gems/passageidentity.

If this is out of scope for you in this PR we can create a bug ticket.

Reply:
hmm i'm not sure why this would be. our [latest published version](https://github.com/passageidentity/passage-ruby/blob/v0.6.2/passageidentity.gemspec#L12) looks like the [ruby-jwt gemspec](https://github.com/jwt/ruby-jwt/blob/main/ruby-jwt.gemspec#L16), whose [homepage](https://rubygems.org/gems/jwt/versions/1.5.4?locale=en) link does work.

it also seems to follow with the [specification](https://guides.rubygems.org/specification-reference/#homepage).

i've deleted the homepage_uri metadata field in cf9ac2affcf1510f3abac469c05dd3e08a959569 either way though.